packages/big-design/src/components/Input/styled.tsx

```diff
@@ -68,47 +69,92 @@ export const StyledInput = styled.input<InputProps>`
     font-size: ${({ theme }) => theme.typography.fontSize.medium};
   }
 
-  ${props =>
-    props.iconRight &&
+  &:-webkit-autofill {
+    background-color: ${({ theme }) => theme.colors.primary10} !important;
+    -webkit-box-shadow: 0 0 0px 1000px ${({ theme }) => theme.colors.primary10} inset;
```

Comment:
This throws a warning on `ts-styled-plugin` 🤔 is there a way to ignore it?

Reply:
I'll 👀 to see if there's a way

Reply:
Unfortunately there isn't a way to disable it on a per-line basis:
https://github.com/Microsoft/typescript-styled-plugin/issues/70#issuecomment-420457682

We could disable linting validation entirely, but I assume we wouldn't want that.